validate.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+import jsonschema
+import yaml
+
+
+def validate(data_filename, schema_filename):
+    # Convert to and from JSON so we can turn date objects into strings
```

Comment:
I'll change this comment to make it more clear. YAML has a "date type" (an unquoted `yyyy-mm-dd`) which is interpreted as a Python `date` object by PyYAML. `jsonschema` gets confused by `date` objects (since it only expects JSON), so we want to convert them back into `yyyy-mm-dd` strings.

Reply:
Actually, since I plan to add an ocflib end-to-end test, I guess it's fine to not do this at all.